tests/conftest.py

```diff
@@ -193,7 +189,7 @@ def firefox_options(
     yield firefox_options
 
     # Delete old pings
-    requests.delete("http://localhost:5000/pings")
+    requests.delete("http://ping-server:5000/pings")
```

Comment:
Same `:5000` question her (and L248 below) as previous comment on L147, re: `os.environ.get("PORT", 5000)` usage from ping_server.py#L48.